swarm/network/stream/delivery.go

```diff
@@ -177,7 +177,7 @@ func (d *Delivery) handleChunkDeliveryMsg(ctx context.Context, sp *Peer, req int
 				// we removed this log because it spams the logs
 				// TODO: Enable this log line
 				// log.Warn("invalid chunk delivered", "peer", sp.ID(), "chunk", msg.Addr, )
-				msg.peer.Drop()
+				msg.peer.bzzPeer.Drop()
```

Comment:
Why do we have to change this? I think you are changing way too many places from peer to bzzPeer, which is essentially the same.